server/middleware/auth.js

```diff
@@ -0,0 +1,19 @@
+const jwt = require('jsonwebtoken');
+const config = require('config');
+
+module.exports = function(req, res, next) {
+  const token = req.header('Authorization');
+
+  if (!token) {
+    return res.status(401).json({ msg: 'No token, Authorization denied' });
```

Comment:
I think this response format would be repeated a lot of times in your codebase. You might want to consider abstracting this logic of generating error messages into one re-useable function

Reply:
No update to this?